### ISSUE TITLE
remove shells and only use templates

### ISSUE THREAD

**Issue by goergen95** [goergen95/flakefast#2]
We can drop into a development shell using flakes via: 

```bash
nix develop "github:goergen95/nix-flakes?dir=templates/r-base"
```

Thus there is no need to duplicate shell/template definitions and keep this in sync.

Downside is that `nix develop "github:goergen95/nix-flakes"` will not work because no default shell (actually none at all) will be defined. However, stacking templates to compose dev environments is the main use case here, so there is not really a need for shells at the main flake level.

**Comment by goergen95** [comment]
Another alternative worth exploring seems to be to import a template flake and inherit its default shell along the following lines:

```nix
let
  r-base = (import ./templates/r-base/flake.nix).outputs {inherit self; inherit nixpkgs;};
in {
  r-base = r-base.devShells.${system}.default;
};
```

**Comment by goergen95** [comment]
Closed in 7763ede0502520fd374a23e1114ecb3e02a35006
